pysb/simulator/bng.py

```diff
@@ -1,12 +1,14 @@
-from pysb.simulator.base import Simulator, SimulationResult, SimulatorException
-from pysb.bng import BngFileInterface, load_equations, generate_hybrid_model
-import numpy as np
+import collections
```

Comment:
In a few files you've got rearranged imports. Unless there's a reason, ideally could you revert this?

Reply:
Done. I found where pycharm was doing it on commits.